meshroom/ui/qml/main.qml

```diff
@@ -902,6 +902,81 @@ ApplicationWindow {
                         property bool updatingStatus: false
                         enabled: !updatingStatus && !_reconstruction.computingLocally
                     }
+                    MaterialToolButton {
+                        id: filePollerRefreshStatus
+                        text: (_reconstruction.filePollerRefresh == 0 || _reconstruction.filePollerRefresh == 2) ? MaterialIcons.sync : MaterialIcons.sync_disabled
+                        ToolTip.text: "Set File Watcher Refresh Status"
+                        ToolTip.visible: hovered
+                        font.pointSize: 11
+                        padding: 2
+                        onClicked: {
+                            refreshFilesMenu.open()
+                        }
+                        enabled: true
+                        Menu {
+                            id: refreshFilesMenu
+                            width: 210
+                            y: parent.height
+                            x: -width + parent.width
+                            MenuItem {
```

Comment:
Would `autoExclusive: true` change the icon?

Reply:
It didn't make a difference: with or without the autoExclusive, the icon is always a state behind unless it is updated "manually" when a MenuItem is selected.